code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1016,6 +1016,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	//metabolism change
 	if(H.nutrition > NUTRITION_LEVEL_FAT)
 		H.metabolism_efficiency = 1
+	if(H.has_trait(TRAIT_DIABETIC))
+		H.metabolism_efficiency = 1.5
```

Comment:
"how do I disguise poison resistance and better medicine as a major negative quirk"

Reply:
Please remember that this makes ALL medicine go out of your system faster, bad and good. So while bad chemicals will get out faster, so will good ones, meaning less healing from them.

Reply:
why is it needed

Reply:
actually efficiency makes medicines last longer, they are an exception

Reply:
cringe

Reply:
Also free GBP for whoever defines the default Metab efficiency then bases these values off define.

Reply:
@V2LenKagamine That is incorrect. This makes toxins and other reagents get purged a lot quicker and "medicine" stays longer. 
`medicine reagents:
holder.remove_reagent(id, metabolization_rate / M.metabolism_efficiency)
toxin reagents and the rest of reagents:
holder.remove_reagent(id, actual_metaboliztion_rate * M.metabolism_efficiency)`
You increased metabolism_efficency, which makes it take less medicine reagents, so they stay longer. The opposite effect for toxins. @Mickyan and @XDTM are correct.